src/lineup/internal/cmds.ts

```diff
@@ -184,15 +184,18 @@ export function setGroupCriteria(provider: IObjectRef<any>, rid: number, columns
 export async function setColumnImpl(inputs: IObjectRef<any>[], parameter: any) {
   const p: LocalDataProvider = await resolveImmediately((await inputs[0].v).data);
   const ranking = p.getRankings()[parameter.rid];
-  const prop = parameter.prop[0].toUpperCase() + parameter.prop.slice(1);
+  let prop = parameter.prop[0].toUpperCase() + parameter.prop.slice(1);
+  if (prop === 'RendererType') {
+    prop = 'Renderer';
+  }
```

Comment:
I saw that we fixed this issue already in the LineUp v4 branch, but did not backport it here. We should try to make the implementation equal to avoid merge conflicts later.

https://github.com/datavisyn/tdp_core/blob/6581047b02e4c5ac1c8f968e64641c0f93716a27/src/lineup/internal/cmds.ts#L215-L227

Please update the implementation in the lineup-v4 branch accordingly.

Reply:
I adapted the implementation to match that of the lineup4 branch. Further tested it  and works as expected (i keep saying that it works as expected and then you find that it doesn't work as expected :smile: )